resources/assets/vue/router/views/dashboard.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+    <div class="flex-center position-ref full-height">
+        <div class="top-right links">
+            <RouterLink :to="{ path: 'countries' }">Countries</RouterLink>
+            <RouterLink :to="{ path: 'currencies' }">Currencies</RouterLink>
+            <RouterLink :to="{ path: 'languages' }">Languages</RouterLink>
+            <RouterLink :to="{ path: 'locales' }">Locales</RouterLink>
+        </div>
+
+        <div class="content">
+            <div class="title">
+                Vertebrae
+            </div>
+
+            <div class="subtitle m-b-md">
+                Built with Laravel {{ laravelVersion }} and Vue {{ vueVersion }}
+            </div>
+
+            <div class="links m-b-md">
+                <a href="https://laravel.com/docs/5.7">Laradocs</a>
+                <a href="https://github.com/nlmenke/vertebrae">GitHub</a>
+            </div>
+
+            <div class="badges">
+                <a href="https://travis-ci.org/nlmenke/vertebrae"><img src="https://travis-ci.org/nlmenke/vertebrae.svg" alt="travis-ci"></a>
+                <a href="https://github.com/nlmenke/vertebrae/blob/master/LICENSE.md"><img src="https://img.shields.io/badge/license-MIT-428F7E.svg" alt="mit-license"></a>
+            </div>
+
+            <div class="contributors">
+                Chiropractors: <a href="https://github.com/nlmenke">nlmenke</a>
```

Comment:
Add yourself to this list? I was thinking of making this a `<ul>`, but I'm not sure how that looks.